text-ui-test/input.txt

```diff
@@ -1,5 +1,12 @@
 start
-help
+tutorial circuit
+template lc
+set v 220
+set c 200
+set l 750
+add parallel l 200
+calc current
+exit
 template rc
```

Comment:
actually this was the old tutorial commands, can remove these then

Reply:
wdym?

Reply:
lines 10 to 16 were the tutorial commands previously. see the `exit` command?

Reply:
we need to check the circuit tutorial right

Reply:
yes I know. but we have 2 sets of tutorial commands for tutorial circuit now.